src/getConfig.ts

```diff
@@ -1,15 +1,22 @@
 import { existsSync, readFileSync } from 'fs'
 import { CONFIG_PATH } from './constants'
-import { printInvalidConfigFile, printConfigFileNowFound } from './print'
+import { printInvalidConfigFile, printConfigFileNotFound } from './print'
 import { Config } from './types'
 
 const getConfig = (): Config => {
   if (!existsSync(CONFIG_PATH)) {
-    printConfigFileNowFound(CONFIG_PATH)
+    printConfigFileNotFound(CONFIG_PATH)
     process.exit(1)
   }
 
-  const config = JSON.parse(readFileSync(CONFIG_PATH, 'utf8').trim())
+  let config
+  
+  try {
+    config = JSON.parse(readFileSync(CONFIG_PATH, 'utf8').trim())
+  } catch (e) {
```

Comment:
I guess the e argument in catch can be removed?

Reply:
Eller om man ska visa upp det på något sätt. Alltid tråkigt när try/catch tyst sväljer fel.

Reply:
Fixade detta i https://github.com/christianalares/earned-clockify-cli/pull/2